addons/website/static/src/js/content/menu.js

```diff
@@ -630,6 +630,22 @@ publicWidget.registry.hoverableDropdown = animations.Animation.extend({
             Dropdown.getOrCreateInstance(toggleEl).hide();
         }
     },
+    /**
+     * @private
+     * @param {Event} ev
+     * @param {String} method bootstrap dropdowns method name
+     */
+    _updateDropdownVisibility: function (ev, method) {
+        const { currentTarget } = ev;
+        if (config.device.size_class <= config.device.SIZES.SM ||
+            currentTarget.classList.contains('o_extra_menu_items')) {
+                return;
+        }
+        const dropdownToggle = currentTarget.querySelector('.dropdown-toggle');
+        if (dropdownToggle) {
+            Dropdown.getOrCreateInstance(dropdownToggle)[method]();
```

Comment:
> but I'm not sure this [line](https://github.com/odoo/odoo/pull/118803/commits/a519dffdc908e95959301a5d4ea4d27ef587086d#diff-875d6d62cea4b05ee632b9c35fbfa92f0f2db831472943a22ef508f0c9662167R646) within guideline or not.

I would personally prefer to receive a boolean parameter and explicitly do `.show()` or `.hide()` according to it but this is ok too.